bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/handler/LeapBridgeHandler.java

```diff
@@ -633,6 +662,26 @@ public void handleMultipleOccupancyGroupDefinition(List<OccupancyGroup> oGroupLi
         }
     }
 
+    @Override
+    public void handleProjectDefinition(Project project) {
+        isRadioRA3 = project.productType.equals("Lutron RadioRA 3 Project");
```

Comment:
```suggestion
        isRadioRA3 = "Lutron RadioRA 3 Project".equals(project.productType);
```

Would also suggest to add a constant with this 'magic string' and add it to the binding constant file, for better maintainability. Not sure if it is used elsewhere, but check that too.